recipe/meta.yaml

```diff
@@ -50,6 +50,7 @@ outputs:
         - zstd
         - simdjson >=3.3.0
       run:
+        - libcurl <8.10
```

Comment:
We are surprised that this is not handled upstream.
See: https://github.com/conda-forge/curl-feedstock/issues/143.